aten/src/ATen/native/cuda/LinearAlgebra.cu

```diff
@@ -50,6 +28,33 @@ Tensor prepare_matrix_for_cublas(Tensor& tensor, bool& transpose_tensor) {
   return tensor_;
 }
 
+Tensor prepare_batch_matrix_for_cublas(Tensor& tensor, bool& transpose_tensor, int64_t& ld_tensor, bool transpose_result, int m, int n) {
+  IntArrayRef tensor_strides = tensor.strides();
+  Tensor tensor_;
+
+  if (tensor_strides[transpose_result ? 2 : 1] == 1 &&
```

Comment:
`transpose_result ? 2 : 1` and `transpose_result ? 1 : 2` is spelled so many times, it would be much clearer if you do
```C++
int fast_dim, leading_dim;
if (transpose_result) {
  fast_dim = 2;
  leading_dim = 1;
} else {
  fast_dim = 1;
  leading_dim = 2;
}
```

Reply:
is there a significance of the names of the variable you suggested?

Reply:
`fast_dim` means the dim that has stride 1, that is, the fast-moving dimension. This is a name from my mind, and feel free to use a better name if you believe there is.